docs/make.jl

```diff
@@ -1,15 +1,15 @@
-using Documenter, NLPModelsKnitro
+using Documenter
 
 makedocs(
-  modules = [NLPModelsKnitro],
+  #modules = [NLPModelsKnitro],
   doctest = true,
-  strict = true,
+  #strict = true,
   format = Documenter.HTML(
     prettyurls = get(ENV, "CI", nothing) == "true",
     assets = ["assets/style.css"],
   ),
   sitename = "NLPModelsKnitro.jl",
-  pages = Any["Home" => "index.md", "Tutorial" => "tutorial.md", "Reference" => "reference.md"],
+  pages = Any["Home" => "index.md", "Tutorial" => "tutorial.md"], # "Reference" => "reference.md"
```

Comment:
It's useful to set preview = true here.

Reply:
Thanks! I added `push_preview = true` in the `deploydocs` function.